pkg/apis/management.cattle.io/v3/tools_system_images.go

```diff
@@ -13,17 +13,17 @@ var (
 		AuthSystemImages     AuthSystemImages
```

Comment:
Can remove the m above. We could remove those lines from mirror.go but that could be disruptive.

Reply:
Removed. Changing the mirror function would impact kdm.